tools/lib/provision.py

```diff
@@ -226,11 +242,10 @@ def main(options):
     # project.
     os.chdir(ZULIP_PATH)
 
-    # setup-apt-repo does an `apt-get update`
     # hash the apt dependencies
     sha_sum = hashlib.sha1()
 
-    for apt_depedency in APT_DEPENDENCIES[codename]:
+    for apt_depedency in SYSTEM_DEPENDENCIES[codename]:
         sha_sum.update(apt_depedency.encode('utf8'))
     # hash the content of setup-apt-repo
     sha_sum.update(open('scripts/lib/setup-apt-repo', 'rb').read())
```

Comment:
Are you thinking we put this into an `if vendor in ["Debian", "Ubuntu"]:` block, and add a `setup-yum-repo` script for RHEL/CentOS, or make `setup-apt-repo` have conditionals for yum (and obviously a new name)?

Reply:
Yes, but there are 2 paths to proceed from here: whether to have `setup-apt-repo` & `setup-yum-repo`, or use a single file `setup-system-repo`.

Reply:
Let's start with 2 scripts; I think it'll be easier to work on / debug.